drivers/note/note_driver.c

```diff
@@ -384,6 +472,69 @@ static ssize_t note_read(FAR struct file *filep, FAR char *buffer,
   return retlen;
 }
 
+/****************************************************************************
+ * Name: note_ioctl
+ ****************************************************************************/
+
+static int note_ioctl(struct file *filep, int cmd, unsigned long arg)
```

Comment:
@YuuichiNakamura could you review PR https://github.com/apache/incubator-nuttx/pull/1731? which make the driver structure more clean.

Reply:
I have reviewed #1731.